practice_app/api/tests.py

```diff
@@ -101,4 +101,37 @@ def test_valid_rows(self):
         self.assertContains(response, 'source')
         self.assertContains(response, 'date')
         self.assertContains(response, 'url')
-        self.assertContains(response, 'position')
\ No newline at end of file
+        self.assertContains(response, 'position')
+class zenodo_test_cases(TestCase):
+
+    def setUp(self):
+        self.c = Client()
+
+    def tearDown(self):
+        print('GET Tests Completed Successfully')
+
+    def test_zenodo_api(self):
+        ACCESS_TOKEN = api_keys.api_keys['zenodo_api']
+        request = requests.get('https://zenodo.org/api/records',
+                               params={'q': 'test', 'sort': 'bestmatch', 'size': 3,
+                                       'access_token': ACCESS_TOKEN})
+        self.assertEquals(request.status_code, 200, "Zenodo API didn't work as supposed to")
+        response = self.c.get("/zenodo/?title=test&rows=3")
```

Comment:
Path in this request seems not valid because it lacks app name in the beginning which is 'api' in our case. It should be changed as:
`response = self.c.get("/api/zenodo/?title=test&rows=3")`